implementations/micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalFxNamingConvention.java

```diff
@@ -106,4 +91,8 @@ public String tagValue(String value) {
         return StringUtils.truncate(formattedValue, TAG_VALUE_MAX_LENGTH);
     }
 
+    private static boolean isAlphabet(char c) {
```

Comment:
Can we use `Character.isLetter` instead?

Reply:
I explicitly decided against that for the reason that it will return true for Unicode letters.

E.g: Character.isLetter('à') will also return true.


Having said that, if we are moving 
> conventionKey = PATTERN_TAG_KEY_DENYLISTED_CHARS.matcher(conventionKey).replaceAll("_");

as for the first condition, I guess this will replace all those with "_" and then I can rely on Character.isLetter().